src/base/static/components/app.tsx

```diff
@@ -301,7 +301,18 @@ class App extends Component<Props, State> {
     this.props.loadPagesConfig(resolvedConfig.pages);
     this.props.loadNavBarConfig(resolvedConfig.nav_bar);
     this.props.loadCustomComponentsConfig(resolvedConfig.custom_components);
-    this.props.loadMapStyle(resolvedConfig.map, resolvedConfig.datasets);
+    this.props.loadMapStyle({
+      mapConfig: resolvedConfig.map,
+      datasetsConfig: resolvedConfig.datasets,
+      analysisTargets: resolvedConfig.place.place_detail
```

Comment:
Here we are pulling out all the mapbox sources that might have analyses run against them, so we can ensure these sources will be available at the time of analysis.

This might simplify when we start reading data from the new forms endpoints.